docs/virtualkeyboard/index.html

```diff
@@ -0,0 +1,455 @@
+              </p>
+            <dt>
+              <dfn>overlaysContent</dfn>
+            </dt>
+            <dd>
+              <p>
+                The attribute enables the author to opt-out of resizing the visual viewport.
+              </p>
+            </dd>
+            <dt>
+              <dfn>boundingRect</dfn>
+            </dt>
+            <dd>
+            <p>
+                The attribute reports the intersection of the VK with the layout viewport in client coordinates.
+            </p>
+            </dd>
+            <dt>
+                <dfn>ongeometrychange</dfn>
+              </dt>
+              <dd>
+              <p>
+                  The event is dispatched when the intersection of the VK and the layout viewport changes, e.g. in response to the VK being shown or hidden or the browser window being repositioned.
+              </p>
+              </dd>
+        </dl>
+    </section>
+    <section data-dfn-for="ElementContentEditable">
+        <h2>Extensions to the <dfn>ElementContentEditable</dfn> mixin</h2>
+        <pre class="idl">
+           partial interface ElementContentEditable {
+            [CEReactions, Reflect, ReflectOnly=("auto","manual")] 
+            attribute DOMString virtualKeyboardPolicy;
+           };
+        </pre>
+        <p>The <dfn><code>virtualKeyboardPolicy</code></dfn> is an <a href=https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute>enumerated attribute</a> whose keywords are the empty string, auto, and manual.
+        <p>When specified on an element that is a contenteditable host, <dfn><code>auto</code></dfn> causes the corresponding editable element to automatically show the VK when it is focused or tapped & <dfn><code>manual</code></dfn> 
+            decouples focus and tap on the editable element from changes in the VK’s current state - the VK remains as it was.
+        </p>
+    </section>
+    <section>
+        <h2>
+          Virtual Keyboard Visibility Change CSS environment variables.
+        </h2>
+        <p>
+          The {{VirtualKeyboard}} API proposes six new <a href="https://drafts.csswg.org/css-env-1/#environment">CSS environment variables</a> that the web developers can utilize to calculate the virtual keyboard's size and adjust layout in a declarative way.
+        </p>
+        <p class="note">
+          These CSS env should be added to the <a href="https://drafts.csswg.org/css-env-1/#env-function">CSS env variable spec</a>.
+        </p>
+        <section>
+            <h3>Keyboard inset variables</h3>
+            <table data-for="env()">
+                <tr>
+                    <th>Name
+                    <th>Value
+                <tr>
+                    <td><dfn>keyboard-inset-top</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
```

Comment:
You probably want to define the semantics of each of these variables and what its value is set to.

Reply:
updated